kcidev/subcommands/detect/__init__.py

```diff
@@ -0,0 +1,39 @@
+import sys
+
+import click
+
+from kcidev.subcommands.detect.issues import issues
+
+
+@click.group(
+    help="""Detect dashboard issues for builds and boots
+
+\b
+Subcommands:
+  issues - Fetch KCIDB issues for builds/boots
+
+\b
+Examples:
+  # Detect build issues
+  kci-dev detect issues --builds --id <build-id>
+  kci-dev detect issues --builds --all-checkouts --days <number-of-days>
+  # Detect boot issues
+  kci-dev detect issues --boots --id <boot-id>
+  kci-dev detect issues --boots --all-checkouts --days <number-of-days>
```

Comment:
I think this would be `kci-dev maestro validate issues` to match the other validation tasks we have.